pubspec.lock

```diff
@@ -3401,5 +3433,5 @@ packages:
     source: hosted
     version: "3.1.2"
 sdks:
-  dart: ">=3.5.0-259.0.dev <4.0.0"
-  flutter: ">=3.19.3"
+  dart: ">=3.5.0 <4.0.0"
+  flutter: ">=3.24.0"
```

Comment:
why we need to change this?

Reply:
this version (3.13.1) of patrol requires min dart sdk version 3.5 which is available from flutter 3.24